provision-osx/tasks/install_xcode.yml

```diff
@@ -55,14 +62,40 @@
 # Check here for versions already installed
 -
   name: Install xcode versions
-  shell: "source ~/.bash_profile && xcversion install {{ item }}"
+  shell: "{{ xcversion_bin.stdout }} install {{ item }}"
   args:
     executable: /bin/bash
   with_items: "{{ xcode_versions }}"
   when: xcode_install_user is defined and xcode_install_password is defined
   register: xcode_install
   failed_when: xcode_cli_tools_install.rc != 0 and xcode_cli_tools_install.stdout.find('already installed') == -1
   changed_when: xcode_cli_tools_install.stdout.find('already installed') == -1
```

Comment:
btw: in two conditions above we should be using `xcode_install` variable. @odra if you agree, could you please add a fix into this PR as well ? Thanks.